docs-source/content/userguide/managing-domains/configoverrides/_index.md

```diff
@@ -366,6 +371,14 @@ Incorrectly formatted override files may be accepted without warnings or errors
       * This directory, or a subdirectory within this directory, should contain each of your custom situational configuration files.
       * If it doesn't, then this likely indicates your domain resource `configOverrides` was not set to match your custom override configuration map name, or that your custom override configuration map does not contain your override files.
 
+* If the Administration Server pod does start but fails to reach ready state:
+  * Check for this message ` WebLogic server failed to start due to missing or invalid situational configuration files` in the Administration Server pod's `kubectl log`
```

Comment:
Since message text can change over time, can you also mention the BEA-XXXXX value that corresponds with this message?

Reply:
That is the new trace message we added to startServer.sh

Reply:
Oops.   I withdraw the comment.
(BTW There's actually a JIRA to number each of our script trace logs...)